### DISCUSSION TITLE
posts/2021/2ff32c0f/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#15] [Announcements]
# posts/2021/2ff32c0f/

在C语言中，循环的实现是基于跳转指令以及状态存储的，对于Lambda演算而言这些是不存在的，那么我们如何实现循环呢？

https://blog.gzti.me/posts/2021/2ff32c0f/

**Comment by youthyJJ** [comment]
写得太棒了, 期待未完待续
